src/pe_reports/pages.py

```diff
@@ -126,125 +138,76 @@ def overview(prs):
         frame = ov_val_05.paragraphs[0]
         run = frame.add_run()
         try:
+            # TODO: Remove hard-coded file locations
+            # Issue 45: https://github.com/cisagov/pe-reports/issues/45
             df_web = pd.read_csv("src/pe_reports/data/csv/dhs_web.csv")
-
-            # TODO: Dry out this code
-            # Issue 18: https://github.com/cisagov/pe-reports/issues/18
-            run.text = str(int(df_web.iloc[0]["count"]))
-            font = run.font
-            Paragraph.text_style_ov_val(font)
+            Pages.add_overview_value(run, df_web)
         except FileNotFoundError as not_found:
             logging.error("%s : There is no customer data.", not_found)
 
         # Bar Graph - Top Level Domains used for masquerading
-        chart = CategoryChartData()
-
-        # TODO: Dry out this code
-        # Issue 18: https://github.com/cisagov/pe-reports/issues/18
-        try:
-            # TODO: Remove hard-coded file locations
-            # Issue 8: https://github.com/cisagov/pe-reports/issues/8
-            tld_df = pd.read_csv("src/pe_reports/data/csv/dhs_tld_df.csv").fillna(0)
-            chart.categories = list(tld_df.columns.values)
-            chart.add_series("Top Level Domains", list(tld_df.loc[0]))
-
-            # TODO: Remove hard-coded graph size and positioning values
-            # Issue 9: https://github.com/cisagov/pe-reports/issues/9
-            x, y, cx, cy = Inches(3.5), Inches(2.9), Inches(2), Inches(1.5)
-            chart = slide.shapes.add_chart(
-                XL_CHART_TYPE.COLUMN_STACKED, x, y, cx, cy, chart
-            ).chart
-            Graph.chart_sm(chart)
-        except FileNotFoundError as not_found:
-            logging.error("%s : There is no customer data.", not_found)
+        # TODO: Remove hard-coded file locations
+        # Issue 45: https://github.com/cisagov/pe-reports/issues/45
+        df_loc = "src/pe_reports/data/csv/dhs_tld_df.csv"
+        col_names = ["Top Level Domains"]
+        chart_type = XL_CHART_TYPE.COLUMN_STACKED
+        size = "small"
+        x, y, cx, cy = Inches(3.5), Inches(2.9), Inches(2), Inches(1.5)
```

Comment:
@aloftus23 - One last question. While we move to the 8.5 x ll format will these x, y, cx, cy values have a set of standards or do you see these as random. 

For example:
half_page = x, y, cx, cy = Inches(5.75), Inches(2.9), Inches(2), Inches(1.5)
quarter_page = x, y, cx, cy = Inches(3.5), Inches(2.9), Inches(2), Inches(1.5)

If this is an unknown right now. I am good with updates.

Reply:
We're still working that out, but that will be the goal. 

One thing to note. Since we are moving to html containers, we only need to set the width and height here; not the location on the page, so it will hopefully be easier to standardize.

Reply:
That works.